packages/contracts/package.json

```diff
@@ -23,14 +24,13 @@
     "build:contracts": "hardhat compile --show-stack-traces",
     "build:contracts:ovm": "hardhat compile --network optimism",
     "build:dump": "ts-node \"bin/take-dump.ts\"",
-    "build:typechain": "hardhat typechain",
+    "build:typechain": "hardhat typechain && hardhat --network optimism typechain",
     "test": "yarn run test:contracts",
     "test:contracts": "hardhat test --show-stack-traces",
     "test:gas": "hardhat test \"test/contracts/OVM/execution/OVM_StateManager.gas-spec.ts\" --no-compile --show-stack-traces",
-    "lint": "yarn run lint:typescript",
-    "lint:typescript": "tslint --format stylish --project .",
-    "lint:fix": "yarn run lint:fix:typescript",
-    "lint:fix:typescript": "prettier --config prettier-config.json --write \"hardhat.config.ts\" \"{src,test}/**/*.ts\"",
+    "lint": "yarn lint:fix && yarn lint:check",
+    "lint:check": "tslint --format stylish --project .",
+    "lint:fix": "prettier --config prettier-config.json --write \"hardhat.config.ts\" \"{src,test,tasks}/**/*.ts\"",
```

Comment:
Noting this changes the behavior of `yarn lint` to be fixing instead of checking.  Do we need to update the CI to make sure it doesn't let some auto-fixable things through the cracks?

Reply:
This is a good point, going to revert this change.

Reply:
Reverted.